src/plugins/maps_legacy/public/map/base_maps_visualization.js

```diff
@@ -61,37 +62,43 @@ export function BaseMapsVisualizationProvider() {
      * @param status
      * @return {Promise}
      */
-    async render(esResponse, visParams) {
+    async render(esResponse = this._esResponse, visParams = this._params) {
+      await this._mapIsLoaded;
+
       if (!this._kibanaMap) {
         //the visualization has been destroyed;
         return;
       }
 
-      await this._mapIsLoaded;
       this._kibanaMap.resize();
```

Comment:
Should we just just call the new `resize` method here?

Reply:
makes sense, done 👍